### DISCUSSION TITLE
How to search on tags

### DISCUSSION THREAD

**Discussion by antmar904** [CrowdStrike/psfalcon#163] [Q&A] [answered]
Hello.
 
I am trying to get a list of hostnames based on tags but I keep getting the returned error of 
"Get-FalconHost: Cannot validate argument on parameter 'Filter'. 'tags' is not a valid property."

**Comment by antmar904** [comment]
`Get-FalconHost "tags:'*AMER*'" -Detailed -All | Select-Object hostname,local_ip,os_version`

**Comment by bk-cs** [comment, marked as answer]
Hi antmar904!

I'm sorry I missed this, and I believe I answered your question already on reddit. This is due to an issue with v2.1.6 and has been corrected in [the v2.1.7 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.1.7).

I will be adding v2.1.7 to the PowerShell Gallery within the next few days, but you can update the module with the release from GitHub to fix it if this continues to be a problem.

**Comment by antmar904** [comment]
Hello. Yes it’s been fix. Great speaking with you and as always thanks for your contributions and help!
